test/index.js

```diff
@@ -192,6 +192,21 @@ test('connect to two peers, append message one send, one note', function (t) {
 
 })
 
+test('replicate a hops=2 peer that my hops=1 friend still doesnt have', function (t) {
+  var state = events.initialize()
+  state = events.clock(state, {})
+
+  state = events.connect(state, {id: 'alice', client: false})
+  state = events.peerClock(state, {id: 'alice', value: {'alice': 1, 'bob': 0}})
```

Comment:
I noticed that this test still works with alice being 0. Not sure what is best, but wanted to at least note that in case someone finds this PR.

Reply:
Ah right

Reply:
I changed it to 0